nyc_taxis/test_procedures/default.json

```
@@ -122,6 +122,34 @@
           {%-if search_clients is defined and search_clients %}
           ,"clients": {{ search_clients | tojson}}
           {%- endif %}
+        },
+        {
+          "operation": "desc_sort_tip_amount",
+          "warmup-iterations": 10,
```

I see rest of the operations have 50 as warmup iterations, can we use the same here?
```suggestion
          "warmup-iterations": 50,
```

---

`10` would be fine for sort queres. But we can make same as we have for others.

---

done.